### ISSUE TITLE
Helm - missing parameters

### ISSUE THREAD

**Issue by milank78git** [apache/camel-k#3763]
Hi, 

I'm using helm the latest Camel-K 1.10.x and  missing parameter KAMEL_OPERATOR_ID 

Installing via binary camel sets this parameter

Please add in operator.yaml helm also :
annotations 
imagePullSecrets 

Thank you very much

**Comment by oscerd** [comment]
You're welcome to open a PR for this.

**Comment by milank78git** [comment]
Thx add PR https://github.com/apache/camel-k/pull/3778/commits/bc3953647964917e694d169ce3f099c783c5662b

**Comment by MehrCurry** [comment]
I am also struggling with the missing KAMEL_OPERATOR_ID

**Comment by MehrCurry** [comment]
My hack workaround is a kyverno policy to patch the variable:

```
---
apiVersion: kyverno.io/v1
kind: ClusterPolicy
metadata:
  name: add-operator-id
  annotations:
    policies.kyverno.io/title: Add Environment Variables KAMEL_OPERATOR_ID
    policies.kyverno.io/subject: Deployment
    policies.kyverno.io/category: Other
    policies.kyverno.io/description: >-
      This i a temporary fix for the missing KAMEL_OPERATOR_ID variable in the
      camel-k helm chart. See https://github.com/apache/camel-k/issues/3763
spec:
  rules:
  - name: add-operator-id
    match:
      resources:
        names:
        - camel-k-operator
        kinds:
        - Deployment
    mutate:
      patchesJson6902: |-
        - op: add
          path: "/spec/template/spec/containers/0/env/-"
          value: {"name":"KAMEL_OPERATOR_ID","value":"camel-k"}
```

**Comment by claudio4j** [comment]
closing as this was fixed in main branch, and released as part of Camel K 1.11.0
